### ISSUE TITLE
Unable to give an argument with a space

### ISSUE THREAD

**Issue by Lucki** [tkashkin/GameHub#194]
I've tried to run [Julius](https://github.com/bvschaik/julius) trough gamehub but Julius complains it can't find the given path.
~~~
INFO: Initializing SDL
INFO: SDL initialized
INFO: Loading game from "/mnt/Games/gamehub/gog/Caesar
INFO: "/mnt/Games/gamehub/gog/Caesar: directory not found
INFO: Exiting: game pre-init failed
~~~
I'm with Julius here because the path doesn't exist but also isn't the path I set in gamehub.
![grafik](https://user-images.githubusercontent.com/1408843/52351156-6bf93d00-2a2a-11e9-81ea-77549080a95e.png)

It works when started in the terminal:
~~~
$ julius "/mnt/Games/gamehub/gog/Caesar 3"
INFO: Initializing SDL
INFO: SDL initialized
INFO: Loading game from /mnt/Games/gamehub/gog/Caesar 3
INFO: Creating screen 1920 x 1080, fullscreen? 1
INFO: Texture created (1920 x 1080)
[…]
~~~

###### Expected behavior
The game starts with the given argument.

###### Actual behavior
It doesn't start, giving the hint it's the wrong argument.

###### Steps to reproduce
Select an executable which wants a single argument. Select an argument with a space.

###### Version and environment

```
Version: 0.13.1-1f713f3-master
Branch:  master
Commit:  1f713f3 (1f713f3f3d13385833106c9fc8a9cfccafc0a52b)
Distro:  Arch Linux
DE:      GNOME
```

Bonus issue:
The user game got combined with the store game and both aren't visible until a restart and the combined listing is still marked as uninstalled despite one of them is installed.

**Comment by neuromancer** [comment]
Can you re-try with the latest revision of the `dev` branch and capture the log using `--debug` ?

**Comment by Lucki** [comment]
Same error with dev branch:
`[DEBUG]  [Utils.run] {'/usr/bin/julius' '"/mnt/Games/gamehub/gog/Caesar' '3"'}; dir: '/usr/bin'`

**Comment by tkashkin** [comment]
Forgot to replace splitting by spaces with an actual parsing in a few places. Should be fixed in eef7ba5.

**Comment by Lucki** [comment]
That worked! :+1:
